From 3da39b57143c93050c59a1180fffb083db89df22 Mon Sep 17 00:00:00 2001
From: Ryan Lortie <desrt@desrt.ca>
Date: Wed, 14 Mar 2012 07:11:56 -0400
Subject: [PATCH] Move the fallback define for GL_PACK_INVERT_MESA

The if-undefined fallback declaration for GL_PACK_INVERT_MESA was
originally added in cogl.c along with code to use it (as part of commit
6f79eb8a5a530fc2fd0a9aed90da172fbf05f742).  Later on, commit
10a38bb14fac3ba8dc05115feb04bde56dc590e7 moved the code that used it to
cogl-framebuffer.c but didn't move the define along with it.  Do that
now.

https://bugzilla.gnome.org/show_bug.cgi?id=672038
---
 cogl/cogl-framebuffer.c |    4 ++++
 cogl/cogl.c             |    4 ----
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/cogl/cogl-framebuffer.c b/cogl/cogl-framebuffer.c
index f18abf1..cdf5f8d 100644
--- a/cogl/cogl-framebuffer.c
+++ b/cogl/cogl-framebuffer.c
@@ -104,6 +104,10 @@
 #ifndef GL_TEXTURE_SAMPLES_IMG
 #define GL_TEXTURE_SAMPLES_IMG            0x9136
 #endif
+#ifndef GL_PACK_INVERT_MESA
+#define GL_PACK_INVERT_MESA 0x8758
+#endif
+
 
 typedef enum {
   _TRY_DEPTH_STENCIL    = 1L<<0,
diff --git a/cogl/cogl.c b/cogl/cogl.c
index 8bc48e8..b60f838 100644
--- a/cogl/cogl.c
+++ b/cogl/cogl.c
@@ -51,10 +51,6 @@
 #include "cogl1-context.h"
 #include "cogl-offscreen.h"
 
-#ifndef GL_PACK_INVERT_MESA
-#define GL_PACK_INVERT_MESA 0x8758
-#endif
-
 #ifdef COGL_GL_DEBUG
 /* GL error to string conversion */
 static const struct {
